vnext/ReactCommon/TEMP_UntilReactCommonUpdate/yoga/yoga/enums/YogaEnums.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include <type_traits>
+
+namespace facebook::yoga {
+
+template <typename EnumT>
+constexpr inline int32_t ordinalCount();
+
+template <typename EnumT>
+constexpr inline int32_t bitCount();
+
+// Polyfill of C++ 23 to_underlying()
+// https://en.cppreference.com/w/cpp/utility/to_underlying
+template <typename T>
```

Comment:
This method was implemented with `auto e` as the parameter upstream, as per the link in the comments, but that was causing an error "a parameter cannot have a type that contains 'auto'" - want to confirm if current change is an appropriate fix.

Reply:
I think this looks okay.